webpack.config.js

```diff
@@ -503,15 +503,28 @@ async function initializeModuleFederation() {
 
             // Other containers will be forced to use the exact versions of shared modules that the web app provides.
             makeSingletonSharedModules([
+                'history',
```

Comment:
Does `history` need to be a singleton dependency? I know we'd want to use the same object everywhere, but since we're getting history from the web app and not from the module itself, I'd think it could be a regular dependency

Reply:
I'm not actually sure if it needs to be singleton, but I know that the Desktop App needs to use the web app's version because we had v5.0.0 loaded in our app and navigation wouldn't work as a result.

Does it matter if it's defined this way?

Reply:
I think it's more idiomatic to not do that, that way module federation can figure out which versions are needed by everything in case they diverge. If having two different versions of the library causes problems, then that needs to be a singleton. It sounds like that's the case here then

Reply:
It causes problems when the versions are different, so let's keep it as a singleton for now. This might be able to change when we upgrade dependencies in the Desktop/Web Apps.